packages/crashlytics/android/src/main/java/io/invertase/firebase/crashlytics/ReactNativeFirebaseCrashlyticsInitProvider.java

```diff
@@ -58,35 +55,12 @@ public String getEmptyProviderAuthority() {
   public boolean onCreate() {
     super.onCreate();
 
-    if (ReactNativeFirebaseCrashlyticsInitProvider.isCrashlyticsCollectionEnabled() && getContext() != null) {
-      ReactNativeFirebaseJSON json = ReactNativeFirebaseJSON.getSharedInstance();
-      boolean useNdk = json.getBooleanValue(KEY_CRASHLYTICS_NDK_ENABLED, true);
-      boolean debug = json.getBooleanValue(KEY_CRASHLYTICS_DEBUG_ENABLED, false);
-
-      try {
-        Fabric.Builder builder = new Fabric.Builder(getContext());
-
-        Crashlytics crashlyticsCore = new Crashlytics.Builder()
-          .core(new CrashlyticsCore.Builder().disabled(!debug && BuildConfig.DEBUG).build())
-          .build();
-
-        if (useNdk) {
-          builder.kits(crashlyticsCore, new CrashlyticsNdk());
-        } else {
-          builder.kits(crashlyticsCore);
-        }
-
-        builder.debuggable(debug);
-
-        Fabric.with(builder.build());
-
-        Log.i(TAG, "initialization successful");
-      } catch (IllegalStateException exception) {
-        Log.e(TAG, "initialization failed", exception);
-        return false;
-      }
-    } else {
-      Log.i(TAG, "auto collection disabled, skipping initialization");
+    try {
+      FirebaseCrashlytics.getInstance().setCrashlyticsCollectionEnabled(ReactNativeFirebaseCrashlyticsInitProvider.isCrashlyticsCollectionEnabled());
```

Comment:
This part I'm not sure about - the previous init seemed to check enabled or not, on in debug or not, and NDK or not, this one is just checking enabled or not. Is this reducing configurability / functionality?

If functionality is being reduced or changed there should be some migration documentation

Reply:
The only documented way to configure NDK is by gradle settings:
https://firebase.google.com/docs/crashlytics/ndk-reports-new-sdk

For debug, is with adb tool :
https://firebase.google.com/docs/crashlytics/test-implementation-new-sdk?platform=android#enable_debug_logging

It seems not configurable anymore in public API.